src/reserves.rs

```diff
@@ -312,33 +343,48 @@ fn challenge_txin(message: &str) -> TxIn {
 #[cfg(test)]
 mod test {
     use super::*;
-    use bdk::bitcoin::secp256k1::ecdsa::{SerializedSignature, Signature};
-    use bdk::bitcoin::{Address, Network, Witness};
-    use bdk::wallet::get_funded_wallet;
+    use bdk_wallet::bitcoin::{Address, Network, Witness};
+    use bdk_wallet::test_utils::get_funded_wallet_single;
+    use bdk_wallet::SignOptions;
     use std::str::FromStr;
 
     #[test]
     fn test_proof() {
         let descriptor = "wpkh(cVpPVruEDdmutPzisEsYvtST1usBR3ntr8pXSyt6D2YYqXRyPcFW)";
-        let (wallet, _, _) = get_funded_wallet(descriptor);
+        let (mut wallet, _) = get_funded_wallet_single(descriptor);
 
         let message = "This belongs to me.";
-        let psbt = wallet.create_proof(message).unwrap();
+        let mut psbt = wallet.create_proof(message).unwrap();
 
         let psbt_b64 = psbt.to_string();
 
-        let expected = r#"cHNidP8BAH4BAAAAAmw1RvG4UzfnSafpx62EPTyha6VslP0Er7n3TxjEpeBeAAAAAAD/////2johM0znoXIXT1lg+ySrvGrtq1IGXPJzpfi/emkV9iIAAAAAAP////8BUMMAAAAAAAAZdqkUn3/QltN+0sDj9/DPySS+70/862iIrAAAAAAAAQEKAAAAAAAAAAABUQEHAAABAR9QwwAAAAAAABYAFOzlJlcQU9qGRUyeBmd56vnRUC5qIgYDKwVYB4vsOGlKhJM9ZZMD4lddrn6RaFkRRUEVv9ZEh+ME7OUmVwAA"#;
+        let expected = r#"cHNidP8BAH4BAAAAAmw1RvG4UzfnSafpx62EPTyha6VslP0Er7n3TxjEpeBeAAAAAAD/////MQvsP2eDTCk3vWfQJ50IOFWLwuTHPsnYikR1hosdK0sAAAAAAP3///8BUMMAAAAAAAAZdqkUn3/QltN+0sDj9/DPySS+70/862iIrAAAAAAAAQEKAAAAAAAAAAABUQEHAAABAR9QwwAAAAAAABYAFOzlJlcQU9qGRUyeBmd56vnRUC5qIgYDKwVYB4vsOGlKhJM9ZZMD4lddrn6RaFkRRUEVv9ZEh+ME7OUmVwAA"#;
+
+        assert_eq!(psbt_b64, expected);
```

Comment:
This is failing because the `Psbt` produced is version 2 while the `expected` `Psbt` value you supplied is version 1. This is coming from [here](https://github.com/bitcoindevkit/bdk/blob/cc15e5d316b0827c3b81e74d586b96c26c174a2e/crates/wallet/src/wallet/mod.rs#L1286C8-L1293C11). Every other data is the same.
![Screenshot 2025-02-10 at 5 53 11 PM](https://github.com/user-attachments/assets/46821a78-e631-487c-8c41-6e1664dc0afe)
![Screenshot 2025-02-10 at 5 53 27 PM](https://github.com/user-attachments/assets/80f97605-36b1-4afe-816f-a74d6dfdb22e)